pages/MainPage.js

```diff
@@ -0,0 +1,35 @@
+export class MainPage {
+
+    constructor(page) {
+        this.page = page;
+        this.title = page.locator('[data-test="title"]');
+        this.cartIcon = page.locator('[data-test="shopping-cart-link"]');
+        this.product = page.locator('[data-test="inventory-item"]');
+        this.productName = page.locator('[data-test="inventory-item-name"]');
+        this.productPrice = page.locator('[data-test="inventory-item-price"]');
+        this.addToCartButton = page.locator('[data-test="add-to-cart-sauce-labs-backpack"]');
+        this.cartBadge = page.locator('[data-test="shopping-cart-badge"]');
+    }
+        
+    async checkProductDisplayed() {
+        const productCount = await this.product.count();
+        if (productCount > 2) { 
+            console.log('Products are shown');
```

Comment:
We dont need methods which logs something to console usually. Lets do some actions (like getProductCount) or have an assertion instead of console.log. And pay attention to naming here `checkProductDisplayed` should check something, but it writes to console (in case log level is set properly), or does nothing in case if caluse resolves to false. So having in mind this logic, we should have this method as `logIfWeHaveMoreThen2Products`. This name sounds dumb, and its the red flag, we have done this method with not reusable/weird logic.